New progeckt/task.css

```
@@ -0,0 +1,36 @@
+h1 { 
+    color:green;
+    font-size: 35px;
+}
+h2 {
+    font-size: 20px;
+    color: green;
+    text-align: left;
+}
+h3 {
+    color: green;
+    text-align: left
+}
+p {
+    text-align: left;
+}
+.life {
+    text-align: right;
+    top: 10px;
+    right: 500px;
+    position: absolute;
+}

```

```suggestion
}

```
для визуального выделения правил